x-pack/plugins/alerting/server/saved_objects/migrations.ts

```diff
@@ -662,6 +662,7 @@ function addRACRuleTypes(
         attributes: {
           ...doc.attributes,
           alertTypeId: ruleTypeMappings[ruleType],
+          enabled: false,
```

Comment:
@elastic/kibana-alerting-services is this sufficient for disabling a rule during migration. Or because the task still exists, the rule runs regardless?

Reply:
As of 8.1, we have this logic in place to skip running rules when they are disabled: https://github.com/elastic/kibana/pull/119239

That hasn't been backported to 8.0 though, so maybe it needs to be to support this PR

Reply:
Even if we backported the above-mentioned PR, I'm wondering if 2 tasks will run when the rule does get enabled. Since we're just setting `enabled` to `false` here but the scheduled task still exists, a new task document will get created when the rule gets enabled and then both tasks will run

Reply:
As part of this PR, we also delete the code that registered the old rule type so the old task still exists but it doesn't get picked up by task manager to actually execute since it can't find the executor logic.

Reply:
Gotcha! Verified that this all works as expected. It will just leave an API key that is never invalidated. Is that a big deal @mikecote ?

Reply:
From a platform perspective, it is ok, it will not cause any harm leaving API keys behind. If folks in the security solution are ok with this caveat, we should be ok.